example/cumulus-tf/main.tf

```diff
@@ -89,6 +89,8 @@ module "cumulus" {
   ems_submit_report     = var.ems_submit_report
   ems_username          = var.ems_username
 
+  es_request_concurrency = var.es_request_concurrency
```

Comment:
Is this variable still used in the cumulus module anywhere if it's not used for dbIndexer?

Reply:
Ah yes, I see that it is